website/docs/terms/data-warehouse.md

```diff
@@ -0,0 +1,84 @@
+
+### Security 
+
+Data warehouses offer advanced security features that ensure your data is always secure. They often directly handle certain compliance strategies needed with healthcare and financial data, eliminating the need for you to do this yourself. They also have features such as roles and users which help you control who has access to your data. But we will get into this more later. 
+
+## Potential business use cases
+
+Businesses can leverage data warehouses for many different reasons. Most of these reasons end up savings time and money for the business, whether directly or indirectly.
+
+### Consolidating all of your data in one place
+
+First, a data warehouse acts as a single source of truth for all of your data. Rather than having all of your data spread across different platforms, it is available to you in one place. This allows you to standardize all of your core metrics and data definitions, rather than depending on the metrics calculated by platforms like Google and Facebook. If you find that different metrics don’t align across platforms, a data warehouse acts as a dependable source for the right metric. Rather than relying on external platforms, you now have one that centralizes all of your data. 
+
+Not to mention, you will save your analytics engineer and data analyst a few headaches. They would otherwise have to manually pull needed data from various sources. Not having a single source of truth decreases your data quality, wastes your data team’s precious time, and makes it difficult to combine data from different sources. 
+
+### Ability to control who has access and the type of access they have
+
+Data warehouses have extensive security features which allow you to control who has access to what. You have the ability to give someone as little or extensive permissions as you wish. Warehouses give you the ability to create users and assign them roles. Each role has its own set of permissions to which databases and <Term id="table">tables</Term> it can see. Then, you can also choose who is allowed to query those tables or even update and delete them. 
+
+When anyone in your org can easily access your data, bad things can happen. You riskthe potential of important data being deleted or incorrectly edited. If you are using Excel spreadsheets for data pulls, this is especially prone to human error. 
+
+### Fast reporting
+
+Because all of your data is located in the same place, it allows for faster reporting compared to pulling data from many different sources. You don’t need to make a bunch of data pulls, upload them to a central location, and then somehow join the datasets. Not to mention the slow speed at which that occurs. A central location allows for you to quickly access and query millions of rows of data, allowing transformations and reporting to be done much faster. 
```

Comment:
```suggestion
Because all of your data is located in the same place, it allows for faster reporting compared to pulling data from many different sources. A central location allows for you to quickly access and query millions of rows of data, allowing transformations and reporting to be done much faster. 
```